packages/bpk-component-accordion/src/BpkAccordionItem.js

```diff
@@ -76,16 +75,15 @@ const BpkAccordionItem = props => {
           </span>
         </button>
       </dt>
-      <AnimateHeight duration={200} height={expanded ? 'auto' : 0}>
```

Comment:
<dl> elements must only directly contain properly-ordered <dt> and <dd> groups, <script> or <template> elements

`<AnimateHeight>` which is a `<div>` must not be a child of `<dl>` only `<dt>`, `<dd>` are allowed to be children of `<dl>`

Functionally still works the same.